src/System.IO.FileSystem.DriveInfo/tests/DriveInfo.Windows.Tests.cs

```diff
@@ -199,7 +199,7 @@ public void VolumeLabelOnNetworkOrCdRom_Throws()
                 {
                     Exception e = Assert.ThrowsAny<Exception>(() => { adrive.VolumeLabel = null; });
                     Assert.True(
-                        e is UnauthorizedAccessException || 
```

Comment:
Are these non-perf test changes necessary? I know some of them are in shared files / base classes, but this one doesn't look necessary. We should try to touch as few files as possible, especially unrelated ones.